cnf-certification-test/operator/suite.go

```diff
@@ -104,3 +112,38 @@ func testOperatorInstallationWithoutPrivileges(env *provider.TestEnvironment) {
 		ginkgo.Fail(fmt.Sprintf("Found %d CSVs with priviledges on some resource names.", n))
 	}
 }
+
+func testOperatorOlmSubscription(env *provider.TestEnvironment) {
+	badCsvs := []string{}
+	if len(env.Csvs) == 0 {
+		ginkgo.Skip("No CSVs to perform test, skipping.")
+	}
+
+	ocpClient := clientsholder.NewClientsHolder()
+	for _, csv := range env.Csvs {
+		ginkgo.By(fmt.Sprintf("Checking OLM subscription for CSV %s (ns %s)", csv.Name, csv.Namespace))
```

Comment:
Not required, add a function to display the CSV, subscription in a helper module